integration-tests/src/test/resources/queries/twitterstream_queries.json

```diff
@@ -19,8 +19,8 @@
                 }
             ],
             "context": {
-                "useCache": "true",
-                "populateCache": "true",
+                "useCache": "false",
```

Comment:
why change this?

Reply:
was trying to see if could allow the tests to pass without setting the new context parameter, but it didn't seem to help, so changed to use the new context parameter.